### DISCUSSION TITLE
Where is the "User" column in RoleModelView? I can't find it in the source code.

### DISCUSSION THREAD

**Discussion by wuqicyber** [apache/superset#30633] [Q&A / Help]
![image](https://github.com/user-attachments/assets/8f4b56ad-cae1-4a7a-84de-b098a514b881)

In the RoleModelView class, we can see:

```python
class RoleModelView:
    list_columns = ["name", "permissions"]
    show_columns = ["name", "permissions"]
    edit_columns = ["name", "permissions"]
    add_columns = edit_columns
```
I don't see a "User" column listed here.
Additionally, it seems that the names in the User Selection Box don't reflect my custom user model.

I'm new to Superset, and I would be grateful if someone could answer my questions:
- Why isn't there a "User" column in the RoleModelView?
- How can I make the User Selection Box use my custom user model?

**Comment by sfirke** [comment]
Have you looked at Flask-AppBuilder? https://github.com/dpgaspar/Flask-AppBuilder  Superset is built on top of FAB and extends its codebase.  I once wanted to add a "last login" field to Superset users page and eventually realized I would have to make that small change in FAB (or introduce a lot of new code into Superset).  I didn't get to it.  Let me know if that helps.

**Reply by wuqicyber** [reply]
The RoleModelView class is part of FAB, and the code I shared is directly from FAB. In Superset, there doesn’t appear to be any override or customization of this class. That’s why I’m unsure where the code responsible for displaying the “User” column is coming from. Could you clarify where this might be implemented or if there’s any additional layer in Superset that I’m missing?
